launcher/InstanceImportTask.cpp

```diff
@@ -409,6 +410,15 @@ void InstanceImportTask::processFlame()
                 {
                     qDebug() << "Will download" << result.url << "to" << path;
                     auto dl = Net::Download::makeFile(result.url, path);
+                    //early signal to catch failures
+                    connect(dl.get(), &Net::Download::failed, this,[&result, &failedFiles](){
+                        //Broken file
+
+                        if(result.url.host() == "media.forgecdn.net"){
```

Comment:
?

Reply:
~~This might have been a push error~~
hemmm this was yes ofc planned for handling blocked files haha